lib/handlers/resourceValidator.js

```diff
@@ -77,7 +77,8 @@ class ResourceValidator {
             const scm = context.scmCredentials;
             this._kuduService = new azure_app_kudu_service_1.Kudu(scm.uri, scm.username, scm.password);
             this._kuduServiceUtil = new KuduServiceUtility_1.KuduServiceUtility(this._kuduService);
-            this._appSettings = yield this.getFunctionappSettingsScm(state, this._kuduService);
+            //Kudu API (GET {publishURL}/api/settings api is not available for kubernetes apps

```

Comment:
Not all changes from src/handlers/resourceValidator are present here.
also lib file for zipdeploy is also missing